koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -35,16 +35,18 @@
 from masu.processor.azure.azure_report_parquet_processor import AzureReportParquetProcessor
 from masu.processor.gcp.gcp_report_parquet_processor import GCPReportParquetProcessor
 from masu.processor.ocp.ocp_report_parquet_processor import OCPReportParquetProcessor
+from masu.util import common as mutil
 from masu.util.aws.common import aws_post_processor
 from masu.util.aws.common import copy_data_to_s3_bucket
 from masu.util.aws.common import get_s3_resource
 from masu.util.aws.common import remove_files_not_in_set_from_s3_bucket
 from masu.util.azure.common import azure_post_processor
-from masu.util.common import get_column_converters
```

Comment:
I liked the explicit imports of the things we needed. Feels like this just added more work to go change tests and all that, but can live with it.

Reply:
Was chasing my tail on this one since the actual error was masked. I thought I needed to qualify with a ns because of a wonky import or something.